### PR TITLE
Color vars

### PR DESCRIPTION
**Pull request by morph027**

### CODE REVIEW

**Comment by enoch85** [comment]
Shouldn't ` <(curl -sL https://cdn.rawgit.com/morph027/vm/color-vars/lib.sh)` be ` <(curl -sL https://cdn.rawgit.com/nextcloud/vm/master/lib.sh)`?

**Comment by morph027** [comment]
Jip. But for testing i used the other branch as the file was not yet inside master. Will fix this in master.

File: nextcloud_install_production.sh [hunk 1]
```diff
@@ -11,9 +11,7 @@
 DEBUG=0
 
 # Repositories
-GITHUB_REPO="https://raw.githubusercontent.com/nextcloud/vm/master"
```
Comment (enoch85) [inline nextcloud_install_production.sh]: I guess you will add all the missing vars in the lib.sh file in the next commit?

**Comment by enoch85** [comment]
> Jip. But for testing i used the other branch as the file was not yet inside master. Will fix this in master.

Great!

File: nextcloud_install_production.sh (hunk 1 above)
Comment (morph027) [inline nextcloud_install_production.sh]: Jip. Should be in there already...

Comment (enoch85) [inline nextcloud_install_production.sh]: Nope, not in your master...

Comment (enoch85) [inline nextcloud_install_production.sh]: Alot is missing actually. Please double check closely.

Comment (enoch85) [inline nextcloud_install_production.sh]: Suggestions, go to all scripts and just copy the vars straight off, then remove the duplicates. This ensures that nothing is missed.

Comment (morph027) [inline nextcloud_install_production.sh]: Sure....right now i was only looking for vars which appears more than one time...

**Comment by morph027** [comment]
Now it's master from my repo, should be updated to original repo when we finally merge stuff.

**Comment by enoch85** [comment]
Ok, do a quick test on a DO VPS. Please mind the stuff to do here as well: https://github.com/morph027/vm/pull/13#issuecomment-291194697

**Comment by enoch85** [comment]
@morph027 I get this in the current master:

```
root@ubuntu-2gb-fra1-01:~# bash nextcloud_install_production.sh 
curl: no URL specified!   
curl: try 'curl --help' or 'curl --manual' for more information
```

**Comment by morph027** [comment]
Hm....looks like ```$NCREPO``` is not set...

**Comment by morph027** [comment]
Will add it...

**Comment by morph027** [comment]
Done

**Comment by enoch85** [comment]
OK thanks, I'll make a PR for the other $vars.

**Comment by morph027** [comment]
:+1: 

Also started on a travis enabled shellcheck ci branch which checks files automatically.

**Pull request merged**